input/fsh/Terminology.fsh

```diff
@@ -83,6 +84,24 @@ Description: "Valueset of the grade of the tumor"
 * SCT#399611001 "High grade histologic differentiation (finding)"
 * NullFlavor#NI "No information"
 
+ValueSet: TNMPrimaryTumorVS
+Id: mcode-tnm-primary-tumor-category-vs
+Title: "TNM Primary Tumor Category Value Set"
+Description: "Valueset of the TNM stage for the T category, according to TNM staging rules"
+* codes from system AJCC
```

Comment:
Can you explicitly enumerate (list out) the AJCC codes here, just like we do with snomed above for example?